cypress/support/pages/common_helper/commonSteps.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
```

Comment:
maybe change the file name from `commonSteps` to probably `globalAssertionSteps` so it will only contains steps in which we will do some sort of assertion then it would give us flexibility in future when we want to create other files like `globalActionsSteps` etc